Thanks both.

@mriscoc:
```
Compiling .pio\build\STM32F103RE_creality_xfer\src\src\lcd\e3v2\proui\plot.cpp.o
Marlin\src\lcd\e3v2\proui\dwin.cpp: In function 'void DWIN_CopySettingsFrom(const char*)':
Marlin\src\lcd\e3v2\proui\dwin.cpp:1848:37: error: 'struct HMI_data_t' has no member named 'CaseLight_Brightness'
 1848 |     caselight.brightness = HMI_data.CaseLight_Brightness;
      |                                     ^~~~~~~~~~~~~~~~~~~~
*** [.pio\build\STM32F103RE_creality_xfer\src\src\lcd\e3v2\proui\dwin.cpp.o] Error 1
```
Due to a missing property in HMI_data_t (that I added) to store CaseLight_Brightness.
Note that I'm also using then `CASE_LIGHT_USE_NEOPIXEL` switch to use the neopixel as a caselight.
The rest of the commit is a simple switch of LED_Color from type `uint32_t` to `LEDColor` to use RGB vs RGBW color macros for colors definitions (eg: `LEDColorWhite()`) .


@thisiskeithb:
You are right, this isn't defined directly, but related to cases like
`#if BOTH(LED_CONTROL_MENU, HAS_COLOR_LEDS)`
So it's better to say: when `NEOPIXEL_LED`, `LED_CONTROL_MENU` and `CASE_LIGHT_USE_NEOPIXEL` are enabled.
I update the issue info.

My config files
[Configurations.zip](https://github.com/MarlinFirmware/Marlin/files/9078043/Configurations.zip)

--- a/Marlin/src/lcd/e3v2/proui/dwin.cpp
+++ b/Marlin/src/lcd/e3v2/proui/dwin.cpp
@@ -2259,7 +2259,11 @@ void SetPID(celsius_t t, heater_id_t h) {
     }
   #endif
   #if HAS_COLOR_LEDS
-    void ApplyLEDColor() { HMI_data.Led_Color = TERN(HAS_WHITE_LED, LEDColor({0, 0, 0, leds.color.w}), LEDColor({leds.color.r, leds.color.g, leds.color.b})); }
+    void ApplyLEDColor() {
+      HMI_data.Led_Color = LEDColor(
+        TERN(HAS_WHITE_LED, { 0, 0, 0, leds.color.w }, { leds.color.r, leds.color.g, leds.color.b })
+      );
+    }
     void LiveLEDColor(uint8_t *color) { *color = MenuData.Value; leds.update(); }
     void LiveLEDColorR() { LiveLEDColor(&leds.color.r); }
     void LiveLEDColorG() { LiveLEDColor(&leds.color.g); }

--- a/Marlin/src/lcd/e3v2/proui/dwin.cpp
+++ b/Marlin/src/lcd/e3v2/proui/dwin.cpp
@@ -1843,11 +1843,6 @@ void DWIN_CopySettingsFrom(const char * const buff) {
   TERN_(PREVENT_COLD_EXTRUSION, ApplyExtMinT());
   feedrate_percentage = 100;
   TERN_(BAUD_RATE_GCODE, HMI_SetBaudRate());
-  #if BOTH(CASE_LIGHT_MENU, CASELIGHT_USES_BRIGHTNESS)
-    // Apply Case light brightness
-    caselight.brightness = HMI_data.CaseLight_Brightness;
-    caselight.update_brightness();
-  #endif
   #if BOTH(LED_CONTROL_MENU, HAS_COLOR_LEDS)
     leds.set_color(
       HMI_data.Led_Color.r,

--- a/Marlin/src/lcd/e3v2/proui/dwin_defines.h
+++ b/Marlin/src/lcd/e3v2/proui/dwin_defines.h
@@ -71,9 +71,6 @@
 #if BOTH(LED_CONTROL_MENU, HAS_COLOR_LEDS)
   #define Def_Leds_Color      LEDColorWhite()
 #endif
-#if ENABLED(CASELIGHT_USES_BRIGHTNESS)
-  #define Def_CaseLight_Brightness 255
-#endif
 
 typedef struct {
   // Color settings
@@ -121,10 +118,6 @@ typedef struct {
   // Led
   #if BOTH(LED_CONTROL_MENU, HAS_COLOR_LEDS)
     LEDColor Led_Color = Def_Leds_Color;
-  #endif
-  // Case Light
-  #if ENABLED(CASELIGHT_USES_BRIGHTNESS)
-    uint8_t CaseLight_Brightness = Def_CaseLight_Brightness;
   #endif
 } HMI_data_t;
 

